crates/solidity/testing/snapshots/cst_output/ContractDefinition/member_user_defined_value_type_definition/generated/0.4.11-failure.yml

```diff
@@ -21,4 +21,10 @@ Tree:
       - ContractKeyword (Token): "contract" # 0..8
       - Identifier (Token): "Sample" # 9..15
       - OpenBrace (Token): "{" # 16..17
-      - SKIPPED (Token): "  type Foo is bool;\n}" # 18..39
+      - ContractMembersList (Rule): # 18..28 "  type Foo"
+          - StateVariableDefinition (Rule): # 18..28 "  type Foo"
+              - TypeName (Rule): # 18..24 "  type"
+                  - IdentifierPath (Rule): # 18..24 "  type"
+                      - Identifier (Token): "type" # 20..24
```

Comment:
I think same issue happens here. `type` is not a valid identifier in this version (or any version?)

Reply:
Yes, they did. Fixed